app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/supportingevidence/do_you_want_to_upload.scala.html

```diff
@@ -17,56 +17,93 @@
-    errorSummary: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.error_summary,
-    formWrapper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
-    yesNo: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.yes_no,
-    caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption_govuk,
-    returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link,
-    backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link
+  layout: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.Layout,
+  errorSummary: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.error_summary_govuk,
+  govukWarningText : GovukWarningText,
+  caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption_govuk,
+  formWrapper: FormWithCSRF,
+  govukRadios: GovukRadios,
+  submitButton: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.submit_button_govuk,
+  returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link_govuk
 )
 
-@(form: Form[Boolean], backLink: Call, isAmend: Boolean, dueRepayment: Boolean = false )(implicit request: RequestWithSessionData[_], messages: Messages, appConfig: ViewConfig)
+@(
+  form: Form[Boolean],
+  backLink: Call,
+  isAmend: Boolean,
+  dueRepayment: Boolean = false
+)(
+  implicit request: RequestWithSessionData[_],
+  messages: Messages
+)
 
-    @key = @{"supporting-evidence.do-you-want-to-upload"}
-    @title = @{messages(s"$key.title")}
-    @hasErrors = @{form.hasErrors || form.hasGlobalErrors}
+@key = @{"supporting-evidence.do-you-want-to-upload"}
 
-    @mainTemplate(title = title, userType = request.userType, hasErrors = hasErrors) {
+@title = @{messages(s"$key.title")}
 
-        @backLinkComponent(backLink)
+@hasErrors = @{form.hasErrors || form.hasGlobalErrors}
 
-        @if(hasErrors) {
-            @errorSummary(form)
-        }
+@layout(
+  title,
+  backLinkUrl = Some(backLink.url),
+  hasErrors = hasErrors
+) {
+  @if(hasErrors) {
+    @errorSummary(form)
+  }
 
-        @caption(messages(s"$key.caption"))
+  @caption(messages(s"$key.caption"))
 
-        @formWrapper(routes.SupportingEvidenceController.doYouWantToUploadSupportingEvidenceSubmit(), 'novalidate -> "novalidate") {
+  <h1 class="govuk-heading-xl">@title</h1>
 
-            @yesNo(
-                fieldId = key,
-                label = pageHeading(title),
-                errorKey = form.error(key).map(e => e.message),
-                hasErrors = hasErrors,
-                selected = form.value,
-                helpText = Some(Html(messages(s"$key.help-text"))),
-                yesContent = None,
-                warningField = if(dueRepayment){Some(messages(s"$key.hint"))}else{None}
-            )
+  <p class="govuk-body-l">@messages(s"$key.help-text")</p>
```

Comment:
using a large font size makes the page appear differently than before - you should use `.govuk-hint` instead and also give the paragraph an id and use that in the fieldset's describedby value because currently this is now less accessible than before when the hint text was associated with the fieldset for screen readers - we can maintain that association by giving the hint text an id and using it in the describedby parameter passed to Fieldset()

Reply:
done